### ISSUE TITLE
UI : Align the chat top bar and the rest of the top bars

### ISSUE THREAD

**Issue by N07070** [Nheko-Reborn/nheko#915]
Hello,

I'm opening this issue to ask if the top bars of Nheko could be aligned. See ; 
![screenshot](https://user-images.githubusercontent.com/2259999/151786777-a9221fcc-71de-4cff-8c34-6b9c451194a0.png)

I propose that the "Spaces" bar and the "Top chat bar" be aligned to the "Chat list bar".

I could do it, and make the PR, if someone indicates to me how to do so.

Cheers,

**Comment by deepbluev7** [comment]
The problem is, that they are not aligned to provide extra functionality. If you remove the second line in the topic, the top bar is aligned with the profile bar. And if you make the tags waste more space, they can be aligned too. I found no way to have it functional as well as aligned. Do you have a good idea?

**Comment by N07070** [comment]
Hum... If we split this issue in two, we might find a way.

Firstly, it should be no issue to align the "Spaces top bar" and the "Account top bar", should it not ? It wastes a few pixels, but it feels overall like a less patched-up UI, in my opinion. Worth it if you ask me. If we look at the Telegram UI, they do it like that. 

Secondly, the "topic tchat top bar" is not aligned, even with only one line in the topic. Maybe making the group name bolder, and the topic a bit smaller would yield a better UX. 

As for a multi-line topics, I see two solutions : One where the topic section expands on a mouse-click, mouse hover or touch action, and one where it's truncated by dots. It's a bit more ambitious. The other solution would be to make that part aligned with the other top bars for one line, and for the multi-line topics, we would be using 1.5 times the space, for a more consistent approach, even if they is no alignment. Otherwise, keeping in mind the Telegram Desktop app, we could remove the topic display in the subtitle, and replace it by the number of users in said group. The group topic is also present in the details of the group, anyways. 

Maybe fixing the first part, which seems low-hanging, and opening another issue for the second part would make more sens ?

**Comment by deepbluev7** [comment]
If I made the tags twice the height, I would lose about 1/3rd of my tags. I guess that is okay, but it feels like such a waste...

![grafik](https://user-images.githubusercontent.com/16312324/151823839-277f4f96-1e81-47da-82e7-54d12144f165.png)

I would probably prefer a 3rd opinion on this, because this might be controversial.

Having the topic show up to 2 lines was pretty highly requested by some people. People use the topic to convey important information and hiding it makes it pretty useless. I guess aligning the single line case would be good in all cases though.

I am really stuck in the function over form case, but if you are willing to put some effort into that, I wouldn't be opposed to review and merge improvements, that improve the look and feel. I might complain, if it affects usability too much though :3

**Comment by N07070** [comment]
> If I made the tags twice the height, I would lose about 1/3rd of my tags. I guess that is okay, but it feels like such a waste...

I'm not sure I was clear ; not making it twice it's heights, but just so the gap ( in red ) in the image below is filled : 


![151823839-277f4f96-1e81-47da-82e7-54d12144f165](https://user-images.githubusercontent.com/2259999/151826997-4fc105cd-4ceb-4845-b919-08e52e9bc9c2.png)

**Comment by N07070** [comment]
> I guess aligning the single line case would be good in all cases though.

Let's focus on this, and see about the rest later ?

**Comment by deepbluev7** [comment]
> I'm not sure I was clear ; not making it twice it's heights, but just so the gap ( in red ) in the image below is filled :

No, I just messed up when typing that. The size would increase by 25% or so, which gives me less space for tags :3
